app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
```

Comment:
now you can translate the text by using i18n then you can ready agin    https://phrase.com/blog/posts/i18n-a-simple-definition/